lib/bin/citgm-all.js

```diff
@@ -217,7 +217,7 @@ function launch() {
       reporter.junit(junit, modules, app.append);
     }
 
-    process.exit(reporter.util.hasFailures(modules));
+    process.exit(+reporter.util.hasFailures(modules));
```

Comment:
nit:
```suggestion
    process.exit(reporter.util.hasFailures(modules) ? 0 : 1);
```
personally i find this much clearer than having to think about boolean number coercion, but ymmv

Reply:
Alright. Applied.